tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "composite": true,
     "incremental": true,
-    "target": "esnext",
+    "target": "es2019",
```

Comment:
ES2020 to pick up optional chaining and nullish coalescing? That's what we're going to ship Lit 2 with